src/libstd/sys/common/wtf8.rs

```diff
@@ -715,6 +715,10 @@ impl<'a> Iterator for Wtf8CodePoints<'a> {
     }
 }
 
+#[doc(hidden)]
+#[unstable(feature = "libstd_sys_internals",
+           reason = "this struct is considered exported for some reason",
+           issue = "0")]
```

Comment:
This is actually intended to be stable (good that this PR caught the bug!), so could you tag it as stable since 1.0.0?